.env.example

```diff
@@ -1 +0,0 @@
-FEEDBACK_EMAIL_ADDRESS=linksf@zendesk.com
```

Comment:
I think .example files are helpful and should be fine, with generalized example data